src/main/java/seedu/address/model/person/Person.java

```diff
@@ -35,10 +36,15 @@ public class Person {
     private final Set<Nric> caregivers = new HashSet<>();
     private final Set<Nric> patients = new HashSet<>();
 
+    // Appointments
+    private final Set<Appointment> appointments = new HashSet<>();
```

Comment:
The comment Appointments/Notes does't seem very insightful? Its the same as the variable name.

Reply:
noted, am removing